mne/viz/tests/test_topomap.py

```diff
@@ -557,6 +557,24 @@ def test_plot_topomap_bads():
     plt.close('all')
 
 
+def test_plot_topomap_channel_distance():
+    """
+    Test topomap plotting with spread out channels (gh-9511, gh-9526).
+
+    Test topomap plotting when the distance between channels is greater than
+    the head radius.
+    """
+    ch_names = ['TP9', 'AF7', 'AF8', 'TP10']
+
+    info = create_info(ch_names, 100, ch_types='eeg')
+    evoked = EvokedArray(np.random.randn(4, 10) * 1e-6, info)
+    ten_five = make_standard_montage("standard_1005")
+    evoked.set_montage(ten_five)
+
+    evoked.plot_topomap(sphere=0.05, res=8)
+    plt.close('all')
```

Comment:
FYI this is no longer necessary as we have a magic autouse mpl fixture that calls `plt.close`

https://github.com/mne-tools/mne-python/blob/2e200aa7778a1462b4da6ce7bf4253b99cde3d6a/mne/conftest.py#L131-L137

Reply:
ok, good, I just followed what other tests there used (and actually thought about a decorator like that), I can change this in the next PR.

Reply:
I am slowly eliminating the ones at the end of functions as I see them. (The ones in the middle of tests are still useful to prevent too many MPL windows from popping up, eating memory, and possibly emitting warnings about 20+ windows.)